machineconfiguration/v1/tests/containerruntimeconfigs.machineconfiguration.openshift.io/DefaultRuntimeConfig.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "ContainerRuntimeConfig"
+crdName: containerruntimeconfigs.machineconfiguration.openshift.io
+tests:
+  onCreate:
```

Comment:
We should also add a test that setting it to an empty string and not providing the field at all acts as expected.